py/picca/delta_extraction/expected_fluxes/dr16_expected_flux.py

```diff
@@ -839,27 +649,14 @@ def compute_var_stats(self, forests):
             var_pipe_bins = var_pipe_bins[w]
 
             # select the wavelength bins
-            if Forest.wave_solution == "log":
-                log_lambda_bins = find_bins(
-                    forest.log_lambda,
-                    self.log_lambda
-                    )
-                # filter the values with a pipeline variance out of range
-                log_lambda_bins = log_lambda_bins[w]
-                # compute overall bin
-                bins = var_pipe_bins + num_var_bins * log_lambda_bins
-            elif Forest.wave_solution == "lin":
-                lambda_bins = find_bins(
-                    forest.lambda_,
-                    self.lambda_,
-                    )
-                # filter the values with a pipeline variance out of range
-                lambda_bins = lambda_bins[w]
-                # compute overall bin
-                bins = var_pipe_bins + num_var_bins * lambda_bins
-            else:
-                raise ExpectedFluxError("Forest.wave_solution must be either "
-                                        "'log' or 'linear'")
+            log_lambda_bins = find_bins(
+                forest.log_lambda,
+                self.log_lambda
```

Comment:
wow, this line is confusing. Are you sure one of these shouldn't be log_lambda_grid?

Reply:
I think this might be right, `self.log_lambda` should be the wavelength bins used for eta/var_lss/fudge whereas forest.log_lambda are the wavelength bins of an individual forest after rebinning to the common grid. But @iprafols should confirm. 
Also according to the docstring self.log_lambda is in restframe, not sure right now about forest.log_lambda. Might be useful to make very clear distinctions in the variable names between rf and obs frame; and between eta-lambda and eg mean_cont lambda

Reply:
I think that would be helpful for the reader. `log_lambda_var_func_grid` would be even more explicit, but also even longer. Similarly I'd recommend renaming the lambda_rest_frame of the mean cont (in case this hasn't a speaking name yet) so that it is always very explicit which lambda exactly is which.

Reply:
I like log_lambda_var_grid

Reply:
I don't think longer names are a problem so I say we go with @Waelthus  suggestion. 

For the mean cont, we use Forest.lambda_rest_frame_grid so I take it we don't need renaming there